chaoslib/litmus/cpu_hog/pod_cpu_hog.yml

```diff
@@ -92,14 +88,14 @@
       ## considering duration in milliseconds from spec
       ## with a grace period of 2 min before cpu stress job terminate
       set_fact:
-        job_wait_time: "{{ ((c_duration|int)/1000 + 120)| int }}"
+        job_wait_time: "{{ ((c_duration|int) + 120)| int }}"
       when: "ramp_time is not defined or ramp_time == ''"
 
     - name: Calculate the total wait time for cpu stress job
       ## considering duration in milliseconds from spec
       ## with a grace period of 2 min before cpu stress job terminate
       set_fact:
-        job_wait_time: "{{ ((c_duration|int)/1000 + (ramp_time|int) + 120)| int }}"
+        job_wait_time: "{{ ((c_duration|int) + (ramp_time|int) + 120)| int }}"
```

Comment:
same here

Reply:
Let's give some name to this `set_fact` task as well

Reply:
pls check the comment at line 95 as well

Reply:
Lets have a separate task for the ramp_time